doc/source/installation.rst

```diff
@@ -0,0 +1,115 @@
+Installation
+============
+
+SIFT is provided either as Conda packages or as a compressed all-in-one
+bundle (tarball/zip) of all dependencies with a single start script.
+Unless otherwise documented below, it is recommended that the software
+bundle be used for any novice users as it requires the least amount of
+prior execution of conda or command line tools. Conda-based installation
+is best for testers, developers, and those wishing to customize versions
+of dependencies used by SIFT.
+
+System Requirements
+-------------------
+
+SIFT works on Windows, Mac, and Linux, but depends on many complex
+libraries which may make installation difficult on some platforms or systems.
+It also makes heavy use of GPU (video card) processing via the PyOpenGL and VisPy
+libraries. In general it is best to have a local physical system to run SIFT as opposed
+to a virtual machine or remote server. Virtual machines and remote servers
+commonly have limited ways of displaying OpenGL visualizations to another
+machine.
+
+While the below system resources should work they should be considered the
+lower end of machines for running SIFT. In general the better GPU and faster
+memory (RAM) the better experience you'll have with SIFT.
+
+* Windows 10+ / Mac OS X >11.0 / Linux >= Rocky Linux 8
+* 8GB RAM
+* Disk space (preferrably on a SSD/NVMe drive) with 20GB+ available
+* GPU with 2GB VRAM and OpenGL 3+ support
```

Comment:
Sorry to reiterate on this. From my last demos at various conferences, one of the first question is always: how will it perform on a simple laptop, without a GPU? My answer was usually: it's ok, as long as you don't load too many layers/time steps/large-array-datasets, as this is my experience on our EUM laptops. I'm just worried that someone looking for the specs and reading about the heavy GPU usage, might decide to not even try if she/he doesn't have one. 

How about pointing out that GPUs are optional, but highly recommended?
```suggestion
* GPU with 2GB VRAM and OpenGL 3+ support (optional, but highly recommended)
```

Reply:
I disagree. Users don't need a monster of a machine and that's fine that we lowered these limits, but we're talking about some high resolution satellite data and you've changed the default image behavior to require loading the entire image into GPU memory. Regardless of what the results of testing were (tiling didn't release memory, etc), this at least logically means that more GPU memory is going to be used. Telling people they don't need a good GPU or that an Intel CPU-integrated GPU is good enough is really risky in my opinion.

Users purchase machines based on these specs. I don't care if someone says "let's see if I can get it to run on my system" for a system they already have and that it works fine on their low-end system. They were going to do that testing anyway. I don't want users to buy new machines, especially if buying multiple for a training course, and have none of those machines work well.

Overall we should do more testing of how things really perform on lower end hardware.

Reply:
Ok, I trust your opinion then :)